realm-transformer/src/main/kotlin/io/realm/transformer/Utils.kt

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2016 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.transformer
+
+import com.android.SdkConstants
+import com.android.build.api.transform.TransformInput
+import javassist.ClassPool
+import javassist.LoaderClassPath
+import java.io.File
+import java.util.jar.JarFile
+
+// Find all classes and append their names to the given String collection
+fun MutableCollection<TransformInput>.appendAllToClassNames(classNames: MutableCollection<String>) {
+    for (item in this) {
```

Comment:
I know that we have type inference for this, but when reviewing code it is still nice if the type is part of the declaration.